### PR TITLE
titan_doc_8.0

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by v01dstar — APPROVED** [review]
LGTM

File: storage-engine/titan-configuration.md [hunk 1]
```diff
@@ -150,9 +150,9 @@ level-merge = false
         discardable-ratio = 1.0
         ```
 
-    > **注意：**
+    > **警告：**
     >
-    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。
+    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。如果磁盘空间不足时强行手动做tikv-ctl compact cluster，可能会导致整个集群无可用空间从而无法写入。
```
Comment (Connor1996) [inline storage-engine/titan-configuration.md:155]:
```suggestion
    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。如果磁盘空间不足时强行手动做 tikv-ctl compact cluster，可能会导致整个集群无可用空间从而无法写入。
```

**Review by Connor1996 — COMMENTED** [review]
blob cache size 相关的描述也顺带改一下？

File: storage-engine/titan-configuration.md [hunk 2]
```diff
@@ -107,6 +107,9 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
 
 通过调整 [`rate-bytes-per-sec`](/tikv-configuration-file.md#rate-bytes-per-sec)，你可以限制 RocksDB compaction 的 I/O 速率，从而在高流量时减少对前台读写性能的影响。
 
+### `shared-blob-cache` v8.0引入
+启用Blob文件和Block文件的共享缓存，默认值为True。Block文件具有更高的优先级。该配置在优先满足Block Cache之后将剩余的缓存用于Blob文件。
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:111]:
```suggestion
你可以通过 [`shared-blob-cache`](/tikv-configuration-file.md#shared-blob-cache-tidb-从-v800-版本开始引入) 控制是否启用 Titan Blob 文件和 RocksDB Block 文件的共享缓存，默认值为 `true`。当开启共享缓存时，Block 文件具有更高的优先级，TiKV 将优先满足 Block 文件的缓存需求，然后将剩余的缓存用于 Blob 文件。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1664,6 +1664,11 @@ rocksdb defaultcf titan 相关的配置项。
 + 推荐值：建议在数据库稳定运行后，根据监控把 RocksDB block cache (`storage.block-cache.capacity`) 设置为能刚好维持接近 95% 以上的 Block Cache 命中率，`blob-cache-size` 设置为 `内存大小 * 50% 再减去 block cache 的大小`。这是为了保证 block cache 足够缓存整个 RocksDB 的前提下，blob cache 尽量大。但 Blob cache 的值不应该设置过大，否则会导致 block cache 命中率大幅下降。
 + 单位：KB|MB|GB
 
+### `shared-blob-cache` TiDB v8.0引入
```
Comment (qiancai) [inline tikv-configuration-file.md:1667]:
```suggestion
### `shared-blob-cache`（从 v8.0.0 版本开始引入）
```

File: storage-engine/titan-configuration.md [hunk 4]
```diff
@@ -150,9 +153,9 @@ level-merge = false
         discardable-ratio = 1.0
         ```
 
-    > **注意：**
+    > **警告：**
     >
-    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。
+    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。如果磁盘空间不足时强行手动做 tikv-ctl compact cluster，可能会导致整个集群无可用空间从而无法写入。
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:158]:
```suggestion
    > 在磁盘空间不足以同时保持 Titan 和 RocksDB 数据时，应该使用 [`discardable-ratio`](/tikv-configuration-file.md#discardable-ratio) 的默认值 `0.5`。一般来说，如果磁盘可用空间小于 50% 时，推荐使用默认值。因为当 `discardable-ratio = 1.0` 时，RocksDB 数据一方面在不断增加，同时 Titan 原有的 blob 文件回收需要该文件所有数据都迁移至 RocksDB 才会发生，这个过程会比较缓慢。如果磁盘空间足够大，设置 `discardable-ratio = 1.0` 可以减小 compaction 过程中 Blob 文件自身的 GC，从而节省带宽。如果在磁盘空间不足时强行执行 `tikv-ctl compact cluster`，可能会导致整个集群无可用空间从而无法写入数据。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1664,6 +1664,11 @@ rocksdb defaultcf titan 相关的配置项。
...
+
++ Blob文件和RocksDB Block文件共享Cache
++ 默认值: True。启用Blob文件和RocksDB Block文件的共享缓存，Block文件具有更高的优先级。该配置在优先满足Block Cache之后将剩余的缓存用于Blob文件。
```
Comment (qiancai) [inline tikv-configuration-file.md:1670]:
```suggestion
+ 是否启用 Titan Blob 文件和 RocksDB Block 文件的共享缓存
+ 默认值：`true`。当开启共享缓存时，Block 文件具有更高的优先级，TiKV 将优先满足 Block 文件的缓存需求，然后将剩余的缓存用于 Blob 文件。
```

File: storage-engine/titan-configuration.md [hunk 6]
```diff
@@ -150,9 +153,9 @@ level-merge = false
...
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:159]: “如果在磁盘空间不足时强行执行 `tikv-ctl compact cluster`，可能会导致整个集群无可用空间从而无法写入数据。” 在这个句话可以挪到 L160 后面吗，这样更醒目

File: storage-engine/titan-configuration.md [hunk 7]
```diff
@@ -107,6 +107,10 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
 
 通过调整 [`rate-bytes-per-sec`](/tikv-configuration-file.md#rate-bytes-per-sec)，你可以限制 RocksDB compaction 的 I/O 速率，从而在高流量时减少对前台读写性能的影响。
 
+### `shared-blob-cache`（从 v8.0.0 版本开始引入）
+
+你可以通过 [`shared-blob-cache`](/tikv-configuration-file.md#shared-blob-cache-tidb-从-v800-版本开始引入) 控制是否启用 Titan Blob 文件和 RocksDB Block 文件的共享缓存，默认值为 `true`。当开启共享缓存时，Block 文件具有更高的优先级，TiKV 将优先满足 Block 文件的缓存需求，然后将剩余的缓存用于 Blob 文件。
+
 ### Titan 配置文件示例
 
 下面是一个 Titan 配置文件的样例，更多的参数说明，请参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。你可以使用 TiUP [修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)，也可以通过[在 Kubernetes 中配置 TiDB 集群](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/configure-a-tidb-cluster)修改配置参数。
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:116]: 1. 这个配置文件示例里要加下 `shared-blob-cache` 吗

image

2. 如果配置了 `shared-blob-cache`， 通常情况是不是就不用配置了 `blob-cache-size` 了

无需再单独配置 [`blob-cache-size`]

**Comment by v01dstar** [comment]
Do not merge this, until https://github.com/tikv/tikv/pull/16460 and https://github.com/tikv/tikv/pull/16517 are merged. If those 2 PRs do not get merged into 8.0 then we should add the changes' documentation to a later release.

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1661,9 +1661,14 @@ rocksdb defaultcf titan 相关的配置项。
 + Blob 文件的 cache 大小。
 + 默认值：0GB
 + 最小值：0
-+ 推荐值：建议在数据库稳定运行后，根据监控把 RocksDB block cache (`storage.block-cache.capacity`) 设置为能刚好维持接近 95% 以上的 Block Cache 命中率，`blob-cache-size` 设置为 `内存大小 * 50% 再减去 block cache 的大小`。这是为了保证 block cache 足够缓存整个 RocksDB 的前提下，blob cache 尽量大。但 Blob cache 的值不应该设置过大，否则会导致 block cache 命中率大幅下降。
++ 推荐值：0。从v8.0.0 之后引入了`shared-blob-cache`，无需单独设置`blob-cache-size`。只有当`shared-blob-cache`没有打开时`blob-cache-size`设置才能生效。
```
Comment (qiancai) [inline tikv-configuration-file.md:1664]:
```suggestion
+ 推荐值：0。从 v8.0.0 开始，TiKV 引入了 `shared-blob-cache` 配置项并默认开启，因此无需再单独设置 `blob-cache-size`。只有当 `shared-blob-cache` 设置为 `false` 时， `blob-cache-size` 的设置才能生效。
```

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1661,9 +1661,14 @@ rocksdb defaultcf titan 相关的配置项。
 + Blob 文件的 cache 大小。
 + 默认值：0GB
 + 最小值：0
-+ 推荐值：建议在数据库稳定运行后，根据监控把 RocksDB block cache (`storage.block-cache.capacity`) 设置为能刚好维持接近 95% 以上的 Block Cache 命中率，`blob-cache-size` 设置为 `内存大小 * 50% 再减去 block cache 的大小`。这是为了保证 block cache 足够缓存整个 RocksDB 的前提下，blob cache 尽量大。但 Blob cache 的值不应该设置过大，否则会导致 block cache 命中率大幅下降。
++ 推荐值：0。从 v8.0.0 开始，TiKV 引入了 `shared-blob-cache` 配置项并默认开启，因此无需再单独设置 `blob-cache-size`。只有当 `shared-blob-cache` 设置为 `false` 时， `blob-cache-size` 的设置才能生效。
```
Comment (qiancai) [inline tikv-configuration-file.md:1664]:
```suggestion
+ 推荐值：0。从 v8.0.0 开始，TiKV 引入了 `shared-blob-cache` 配置项并默认开启，因此无需再单独设置 `blob-cache-size`。只有当 `shared-blob-cache` 设置为 `false` 时，`blob-cache-size` 的设置才能生效。
```

File: storage-engine/titan-configuration.md [hunk 10]
```diff
@@ -150,12 +153,16 @@ level-merge = false
         discardable-ratio = 1.0
         ```
 
-    > **注意：**
+    > **警告：**
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:156]:
```suggestion
    > **注意：**
```

**Review by tonyxuqqi — COMMENTED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

File: storage-engine/titan-configuration.md [hunk 11]
```diff
@@ -107,6 +107,10 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
 
 通过调整 [`rate-bytes-per-sec`](/tikv-configuration-file.md#rate-bytes-per-sec)，你可以限制 RocksDB compaction 的 I/O 速率，从而在高流量时减少对前台读写性能的影响。
 
+### `shared-blob-cache`（从 v8.0.0 版本开始引入）
+
+你可以通过 [`shared-blob-cache`](/tikv-configuration-file.md#shared-blob-cache-tidb-从-v800-版本开始引入) 控制是否启用 Titan Blob 文件和 RocksDB Block 文件的共享缓存，默认值为 `true`。当开启共享缓存时，Block 文件具有更高的优先级，TiKV 将优先满足 Block 文件的缓存需求，然后将剩余的缓存用于 Blob 文件。
```
Comment (qiancai) [inline storage-engine/titan-configuration.md:112]:
```suggestion
你可以通过 [`shared-blob-cache`](//tikv-configuration-file.md#shared-blob-cache从-v800-版本开始引入) 控制是否启用 Titan Blob 文件和 RocksDB Block 文件的共享缓存，默认值为 `true`。当开启共享缓存时，Block 文件具有更高的优先级，TiKV 将优先满足 Block 文件的缓存需求，然后将剩余的缓存用于 Blob 文件。
```

**Pull request merged**